service/worker/batcher/activities.go

```diff
@@ -308,3 +332,98 @@ func isDone(ctx context.Context) bool {
 		return false
 	}
 }
+
+func getResetEventIDByType(ctx context.Context,
```

Comment:
plz add some tests for this logic

Reply:
@yycptt Just added tests for this, please take a look. Thanks